src/digitalIdentity-frontend/src/app/pages/home-page/home-page.component.css

```diff
@@ -58,3 +60,25 @@ a {
   display: flex;
   justify-content: space-between;
 }
+
+.icon {
+  color: 500;
+  transform: scale(4);
+}
+
+.arrow {
+  color: 500;
```

Comment:
Please just to know! What does that mean? What color is it exactly? I have the impression that it is not applied to the page.

Reply:
This are the colours of adorsys in the styles.scss

![image](https://user-images.githubusercontent.com/93184461/180429843-89afbf6d-2ab2-45f0-a7a9-bd3cb3420df6.png)

Reply:
The easiest way to solve this is to put the color definition in the style.scss file and to add a class to the icons so that you don't break the color of other icons
=> 
home-page.component.css:
.icon {
  transform: scale(4);
}

.arrow {
  transform: scale(3);
}

styles.scss:
.home-page-icons {
  color: $adorsys-primary
}

I hope this might help :)

Reply:
Okay nevermind, Annika told me a much easier way. She also wrote it in the comments of the style sheet somewhere:
You can add color="primary" in the HTML tag of all material components you're using.